.gitignore

```diff
@@ -19,12 +19,15 @@
 .env.test.local
 .env.production.local
 .cache
+.config
+.bash_history
```

Comment:
Errr, I don't think we want `.bash_history` in case a password is used in the CLI?